src/apps/filters/h3.c

```diff
@@ -121,12 +162,21 @@ bool generalHelp(int argc, char *argv[]) {
         .helpText =
             "Convert degrees latitude/longitude coordinate to an H3 cell.",
     };
-    Arg *args[] = {&helpArg, &cellToLatLngArg, &latLngToCellArg};
+    Arg cellToBoundaryArg = {
+        .names = {"cellToBoundary"},
+        .helpText =
+            "Convert an H3 cell to an array of latitude/longitude coordinates "
+            "defining its boundary",
+    };
```

Comment:
Nit: Is it possible to define this a top-level `static const` so that we can reuse it instead of repeating it?

Reply:
See my prior comment. Leaving DRY-ing it to another PR focused solely on that so we can better debate the merits/long-term maintenance.